node/builder.go

```diff
@@ -404,12 +406,40 @@ func ConfigBoost(c interface{}) Option {
 		Override(new(*sectorblocks.SectorBlocks), sectorblocks.NewSectorBlocks),
 
 		// Sealing Pipeline State API
-		Override(new(sealingpipeline.State), From(new(modules.MinerStorageService))),
+		Override(new(sealingpipeline.API), From(new(modules.MinerStorageService))),
 
 		Override(new(*storagemarket.Provider), modules.NewStorageMarketProvider(walletMiner)),
 
 		// GraphQL server
 		Override(new(*gql.Server), modules.NewGraphqlServer),
+
+		// Config (todo: get a real property system)
```

Comment:
What do you mean by a "real property system"?

Reply:
Copied over from Lotus. The idea is that we don't want a new set of overrides for every single config, but rather a dynamic property system, where we can add properties and hot-reload / change them, without having to statically define every single one of them.

Reply:
Ok makes sense - could we update the comment to clarify 🙏

Reply:
I removed the comment altogether. We will probably get rid of these dynamic configs soon.